common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1710,6 +1712,8 @@ public String toString() {
 
   /** Metadata tracked for an actively merged shuffle partition */
   public static class AppShufflePartitionInfo {
+
+    private static final Cleaner CLEANER = Cleaner.create();
```

Comment:
This is still within `AppShufflePartitionInfo`, see more [here](https://github.com/apache/spark/pull/43371#discussion_r1359142068).

Reply:
Sorry, it was my mistake. It has been corrected.